experimental/sql/src/impl_to_iterators.py

```diff
@@ -30,6 +32,15 @@ def convert_datatype(self, type_: RelImpl.DataType) -> Attribute:
     if isinstance(type_, RelImpl.Int64):
       # TODO: this is obviously broken, but the backend currently only support i32
       return IntegerType.from_width(32)
+    if isinstance(type_, RelImpl.Decimal):
```

Comment:
I guess that the following comment is really about some earlier PR: You currently only support a static number of digits before and after the point, right? How many? You need to be careful about doing math with those: some operations, such as multiplication, change the number of required digits (e.g., `0.01 * 0.01 == 0.0001`). If you parametrized the type with the number of digits after the point, you could do "math on types" in order to reflect that. How do you plan to deal with it currently?

Depending on what you do, I suggest you add comments about shortcuts being taken (which is absolutely fine!) and/or rename the type to `DecimalP8S2` (for "decimal with a *precision* of 8 decimal digits and a *scale* of two decimal digits") or similar. (See also the [T-SQL decimal type](https://docs.microsoft.com/en-us/sql/t-sql/data-types/decimal-and-numeric-transact-sql?view=sql-server-ver16) for example.)

Reply:
I plan to add two parameters to decimal in a subsequent PR. The thing about arithmetic on them is interesting tough. I will need to think about it and investigate what ibis does exactly. I might push a version of that with the parametrization. Added a comment in the BinOp rewrite.